src/components/Link/Link.scss

```diff
@@ -1,33 +1,26 @@
 .Link {
-    color: var(--color-brand-50);
-    text-decoration: var(--link-decoration-normal);
-    transition: color var(--transition-duration);
+    color: var(--color-link-default);
+    text-decoration: underline;
 
-    &:hover,
     &:focus {
-        color: var(--color-brand-60);
-        text-decoration: var(--link-decoration-hover);
+        outline: 2px solid var(--color-border-information-subtle-default, #66AFFF);
```

Comment:
I learned today about `hover-visible` (thank you 🙏) and It seems to be a perfect place to use it instead of just `:focus` here. 
This way, we can avoid showing an outline on mouse clicks and only display it during keyboard interactions as specified in the ticket.